rfcs/text/0031-risk-fields.md

```diff
@@ -21,6 +21,22 @@ Stage X: Provide a brief explanation of why the proposal is being marked as aban
 <!--
 Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting, and add them to the corresponding RFC folder.
 -->
+The `risk` fields being proposed are as follows:
+
+Field | Type | Example | Description | Use Case
+-- | -- | -- | -- | -- 
+risk.calculated_score_norm | float | 88.73 | A risk classifications core (0-100) calculates by an internal system as part of entity analytics and entity risk scoring | Can be used to indicate the risk associated with a particular host
+risk.calculated_level | keyword | High | A risk classification level calculated by an internal system as part of entity analytics and entity risk scoring | Can be used to indicate the risk associated with a particular host
+risk.static_score_norm | float | 83.0 | A risk classification score (0-100) obtained from outside the system, such as from some external Threat Intelligence Platform | Can be used to indicate the projected risk of a particular host based on a trusted third party intelligence feed 
+risk.static_level | keyword | High | A risk classification level obtained from outside the system, such as from some external Threat Intelligence Platform | Can be used to indicate the projected risk of a particular host based on a trusted third party intelligence feed
+risk.factors| object | See Source data section | Factors that contributed to the risk | Explainability about what contributed to the risk
```

Comment:
I am concerned here about not having a defined structure for these objects. Do you think that's something we could add?

Reply:
@kgeller Wdym by not having a defined structure for the objects? I have also added definitions for `risk.factors.alerts` and `risk.factors.others`. So essentially, the `risk.factors` object can only further have the fields `alerts` and `others` . Furthermore, I have also tried to specify how these fields should look like. For example `risk.factors.alerts` should be an array of objects. If there's a preferred way to elaborate on these to maybe specify the structure of the objects themselves, I am happy to do so- I just wasn't sure where and how to add that.

Reply:
Looks awesome! Thanks for getting those added in